sound/soc/sof/sof-priv.h

```diff
@@ -295,7 +295,7 @@ struct snd_sof_pcm {
 	struct snd_sof_pcm_stream stream[2];
 	struct list_head list;	/* list in sdev pcm list */
 	struct snd_pcm_hw_params params[2];
-	int hw_params_upon_resume[2]; /* set up hw_params upon resume */
+	int prepared[2]; /* PCM_PARAMS set successfully */
```

Comment:
@kv2019i bool

Reply:
@lyakh I got earlier comment from Pierre that bool should not be used in structs (even if not packed) and I've stopped adding them to structs. Coding style doc seems to now allow them. I also see you now added one snd_sof_dev, but it's really quite messy now when we use 'int', 'u32' and 'bool' for booleans. Is "bool" now ok in structs as well?